docs/en/operation/Troubleshooting.md

```diff
@@ -191,19 +178,6 @@ the property `alluxio.master.journal.folder` setting. There can be multiple reas
 some HDFS datanodes serving the journal files are under heavy load or running out of disk space. Please ensure the
 HDFS deployment is connected and healthy for Alluxio to store journals when the journal directory is set to be in HDFS.
 
-### Q: I'm seeing that client connection was rejected by master
-
-A: When you see errors from applications like `"alluxio.exception.status.UnavailableException:
-Failed to connect to BlockMasterClient @ hostname:19998 after 13 attempts"`, a possibility is the
-Alluxio master server has run out threads in its thread pool to serve new incoming client requests.
```

Comment:
@bf8086 do we possibly still run into this issue? Any suggestion here after `alluxio.master.worker.threads.max` was removed?

Reply:
It should be less likely for user to run into this issue in 2.0 because of the new shared thread model, but users can tweak `alluxio.master.rpc.executor.parallelism` if they see delays or timeout on RPC request. @ggezer WDYT?

Reply:
@bf8086 @ggezer  if you think it is possible, can you contribute a paragraph on this in a separate doc fix pr?